LOCAL_DEVELOPMENT.md

```diff
@@ -0,0 +1,15 @@
+# Local Development
+
+To be able to run the app on your machine, do the following:
+
+1. Create a `.env` file with the right values (ask for it in slack)
+2. You can use a database specifically for dev (local) development in Digital Ocean
+   you can find it here https://cloud.digitalocean.com/databases/17c20f28-ea8f-40fb-8d10-942be051789f?i=afde57
+   get the connection url string and set it in you .env file as (redacted)
+   `DATABASE_URL=postgresql://user:password@name.c.db.ondigitalocean.com:25061/local?sslmode=require`
+   if you wish to set up using a local database, create one anyway you want (docker is easier) and use the connection
+   string for that instead. Keep in mind you will need to migrate it, the easiest way to do it is by
+   running `docker-compose build calcom` as migrating the db is part of the image build (for some reason idk).
```

Comment:
IIRC it is part of it because it was required to get the docker image running (its been a while since that was dockerized so I don´t recall exactly)
That being said, I think there is also a missing step in there, we had to run a migration on the appstore or something, in order for google meet and calendar integrations to work
For whatever reason instead of storing the API keys needed in the env file, they store them in DB
So in order to run locally and have things actually work as expected, we need to do that
we also need to do it for the Dev DB in all likelihood as well

Reply:
pushed